packages/bpk-component-segmented-control/src/BpkSegmentedControl-test.tsx

```diff
@@ -0,0 +1,102 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {  render, fireEvent, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+import BpkSegmentedControl from './BpkSegmentedControl';
+import SEGMENT_TYPES from './segmentTypes';
+
+const mockOnItemClick = jest.fn();
+
+const defaultProps = {
+  buttonContents: ['one', 'two'],
+  onItemClick: mockOnItemClick,
+  selectedIndex: 1,
+  shadow: false,
+  type: SEGMENT_TYPES.CanvasContrast,
+};
+
+describe('BpkSegmentedControl', () => {
+  beforeEach(() => {
+    mockOnItemClick.mockClear();
+  });
+
+  it('should render component correctly', () => {
+    const { asFragment } = render(<BpkSegmentedControl {...defaultProps} />);
+
+    expect(asFragment()).toMatchSnapshot();
```

Comment:
We don't use snapshots anymore for testing as they are unreliable so can this be changed to be tested in other methods

Reply:
Do you have a suggestion on how you would test the 'should render correctly'? 
Checking the codebase and I can see 30+ examples of `.toMatchSnapshot();` in use, and none using an alternative.

Reply:
So here I would have a test that would just simply look to see if certain things are rendered like the test below on L45!

We are working towards moving away from snapshots, but its an over time migration vs just doing all at once